app/services/open_library/book_search.rb

```diff
@@ -0,0 +1,15 @@
+module OpenLibrary
+  class BookSearch < Base
+    def self.execute(number_isbn)
```

Comment:
Beware of the English
```suggestion
    def self.execute(isbn_number)
```